cmd/generate_completion_documentation.go

```diff
@@ -65,7 +65,7 @@ func main() {
 		if generators.InstallOnlyMode() {
 			return
 		}
-		updateFromCmd(commands, filepath.Join("doc", "14.-Command-line-reference.md"))
+		updateFromCmd(commands, filepath.Join("doc", "15.-Command-line-reference.md"))
```

Comment:
I think a nice gardening activity will be to use globbing to match it :) but not related to that one)